### ISSUE TITLE
Refund action for over/underfunded Beta Asset

### ISSUE THREAD

**Issue by bonomat** [comit-network/comit-rs#1155]
With the latest changes in #1135 we do not consider overfunded HTLCs as funded anymore.
This was a relative easy change for the HTLC on alpha ledger, however, for the HTLC on the beta ledger our current state machine will _explode_ with various new states (AlphaDeployBetaInvalidFunded, AlphaFundedBetaInvalidFunded, AlphaRedeem*, AlphaRefunded*, ...) hence we ( @thomaseizinger  + @bonomat ) decided on just crashing the state machine for now and postpone this implementation when we update our state machine.

**Comment by thomaseizinger** [comment]
Blocked on revising the state machine.

**Comment by bonomat** [comment]
Closed for now. If this _bug_ happens then we will fix it on the spot. 
@thomaseizinger : please put this on your technical debt list ;)
